### ISSUE TITLE
q2pro build bug with CONFIG_PATH_HOME

### ISSUE THREAD

**Issue by darkshade9** [actionquake/distrib#70]
Windows builds don't appear to honor the `CONFIG_PATH_HOME` value, it doesn't even show up in the build process, example:

Windows:
```
i686-w64-mingw32-gcc -c -Werror -std=gnu99 -O2 -Wall -g -MMD -I../q2pro-mgw-sdk/inc -D__USE_MINGW_ANSI_STDIO=1 -iquote./inc -msse2 -mfpmath=sse -DCPUSTRING='"x86"' -DBUILDSTRING='"Win32"' -DREVISION=1 -DVERSION='"r1~5aea4e3"' -DBASEGAME='"action"' -DDEFGAME='"action"' -DUSE_SERVER=1 -DUSE_AC_SERVER=1 -DUSE_MVD_SERVER=1 -DUSE_MVD_CLIENT=1 -DUSE_ZLIB=1  -DUSE_ICMP=1 -DUSE_SYSCON=1 -DUSE_FPS=1 -DUSE_PACKETDUP=1 -DUSE_DBGHELP=1 -DUSE_WINSVC=1 -o .q2proded/src/common/bsp.o src/common/bsp.c
```

Linux:
```
gcc -c -Werror -std=gnu99 -O2 -Wall -g -MMD  -iquote./inc -fvisibility=hidden -DCPUSTRING='"x86_64"' -DBUILDSTRING='"Linux"' -DREVISION=1 -DVERSION='"r1~5aea4e3 STANDALONE"' -DBASEGAME='"action"' -DDEFGAME='"action"' -DDATADIR='"."' -DLIBDIR='"."' -DHOMEDIR='"~/aqtion"' -DUSE_SERVER=1 -DUSE_AC_SERVER=1 -DUSE_MVD_SERVER=1 -DUSE_MVD_CLIENT=1 -DUSE_ZLIB=1  -DUSE_ICMP=1 -DUSE_SYSCON=1 -DUSE_FPS=1 -DUSE_PACKETDUP=1 -o .q2proded/src/common/bsp.o src/common/bsp.c
```

Note the missing `DHOMEDIR` from the Windows side.

I tested this locally by setting the cvar`homedir` to `%LOCALAPPDATA%\.aqtion` and it worked great.  Something in the build process is ignoring this value for Windows builds.

**Comment by darkshade9** [comment]
I assume this is a limitation with Windows somehow, but we can overcome this by setting the `homedir` cvar in Steam.  I don't know of a good resolution in the Standalone

**Comment by darkshade9** [comment]
Fixed in v1.0.6
